### PR TITLE
Feature/db tests fix

### PR DESCRIPTION
**Pull request by TheRealSeber**
Now every time the tests are started, the database will be created. after finishing them it will be deleted. (see confest.py)

### CODE REVIEW

**Review by matined — DISMISSED** [review]

**Comment by matined** [comment]
@KubaKrzych it should solve your problems with CI pipeline ;)

**Review by matined — CHANGES_REQUESTED** [review]

**Comment by KubaKrzych** [comment]
Well, whenever I set up a db with _docker-compose up database_ as a process, and run a _pytest_ command I still get the same error that the schema "core" doesn't exist.

If we only want to test the db, I'm pretty sure I should be able to test the db without creating the containerised api application. @TheRealSeber , to make sure everything works as is supposed to, you can test it locally by running:

1. _docker-compose up database_
2. pytest

There should be no errors after running these two commands. If there are any, it means that something still needs to be done.

File: app/api/tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+engine = sqlalchemy.create_engine(
+    settings.TEST_SQLALCHEMY_DATABASE_URI, pool_pre_ping=True
+)
+
+
+@pytest.fixture(scope="session", autouse=True)
+def setup():
+    # Set up the database
+    models.Base.metadata.drop_all(bind=engine)
```
[the remark this answers is not in the document]
Comment (TheRealSeber) [reply app/api/tests/conftest.py:14]: > @TheRealSeber we miss creating 'core' shema if not exists

no, in the models module there is a code

```
metadata_obj = MetaData(schema="core")
```

which as i know adds metadata data creates schema core. moreove i was testing it locally and it worked, both yesterday and now

[the remark this answers is not in the document]
Comment (matined) [reply app/api/tests/conftest.py:14]: it doesnt work locally, do you remove the old database container when testing? Backend creates the schema using this piece of code: 
image

I think we miss it it tests/ folder

[the remark this answers is not in the document]
Comment (TheRealSeber) [reply app/api/tests/conftest.py:14]: added

**Review by matined — APPROVED** [review]

**Pull request merged**